lib/macho/load_commands.rb

```diff
@@ -741,6 +744,42 @@ def initialize(view, cmd, cmdsize, htoffset, nhints)
       super(view, cmd, cmdsize)
       @htoffset = htoffset
       @nhints = nhints
+      @table = TwolevelHintsTable.new(view, htoffset, nhints)
+    end
+
+    # A representation of the two-level namespace lookup hints table exposed
+    # by a {TwolevelHintsCommand} (`LC_TWOLEVEL_HINTS`).
+    class TwolevelHintsTable
+      # @return [Array<MachO::TwoLevelHintsTable::TwoLevelHint>] all hints in the table
+      attr_reader :hints
+
+      # @param view [MachO::MachOView] the view into the current Mach-O
+      # @param htoffset [Fixnum] the offset of the hints table
+      # @param nhints [Fixnum] the number of two-level hints in the table
+      # @api private
+      def initialize(view, htoffset, nhints)
+        format = (view.endianness == :little) ? "V<" : "V>"
+        raw_table = view.raw_data.slice((htoffset...(htoffset + (nhints * 4))))
+        blobs = raw_table.unpack(format * nhints)
```

Comment:
> So far we've only unpacked structures of fixed length, so I'm not sure if this is a good enough argument.

Yeah, it falls apart when you consider that.

One other possible argument is that we may end up (not here, but in sections/blobs referenced by `dyld` commands) unpacking non-fixed structures whose fields vary in width.

In a case like that, using `*` would cause the unpacker to be excessively greedy and swallow up fields with incorrect widths (for example, `Q=*L=*`). Being explicit about expected elements here sets a good style precedent for more complex cases like the above.